src/components/Process/Process.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+import { PROCESS } from '../ThemeIdentifiers';
+import * as baseTheme from './Process.scss';
+
+export interface NavState {
+  indx: number;
+  styles: string[];
+}
+
+export interface Step {
+  name: string;
+  component: React.ReactNode;
+}
+
+export interface Props {
+  // JSON data source of which we need to indicate process for.
+  steps: Step[];
+  showNavigation?: boolean;
+  // To allow user to enable / disable backstepping on process indicator for navigation.
+  allowBackStepping?: boolean;
+  // To apply any theme to component via themer.
+  theme?: any;
+  // To apply custom styling.
+  componentStyle?: React.CSSProperties;
+    // Custom class
+  componentClass?: string;
+}
+
+export interface State {
+  showPreviousBtn: boolean;
```

Comment:
@HiirenP Always use the full name, like `showPreviousButton` instead of `Btn`, do this change in all the places of this component